Sage/carbon
# chore(i18n): remove classic theme - FE-3111

removal of classic story and redundant docs. Stories have been refactored in to CSF and an MDX
example created.

fixes FE-3111

### Proposed behaviour
- Remove `classic` story
- Refactor stories file to CSF
- Create MDX example for component 
- Remove redundant documentation

### Current behaviour
- `classic` story is present
- Stories file is in the deprecated `storiesOf` format
- Documentation conforms with our previous standards

### Checklist



### Additional context
N/A

### Testing instructions
- `git checkout FE-3111_i18n_classic_theme_removal`
- `npm start` to start Storybook
- Navigate to i18nComponent
- Component should not have a classic example and MDX example should be present.
